config/config.go

```diff
@@ -300,6 +315,20 @@ func (cfg *RPCConfig) ValidateBasic() error {
 	return nil
 }
 
+// returns true if cors is enabled.
+func (cfg *RPCConfig) IsCorsEnabled() bool {
+	return len(cfg.CORSAllowedOrigins) != 0
+}
+
+// returns cors options for RPC endpoint
+func (cfg *RPCConfig) CorsOptions() cors.Options {
```

Comment:
I'd prefer not to create this method because it ties our config API with a third-party library API.